hosts_example.yml

```diff
@@ -165,13 +165,25 @@ kafka_broker:
       ## By default the first host will get broker id=1, second gets id=2. Set broker_id to customize
       # broker_id: 3
 
+      ## (optionally) Rack aware replication assignment for fault tolerance. Examples: `RACK1`, `us-east-1d`
```

Comment:
We should introduce this a little bit. "Properties can also be applied on a host by host basis. In the below example we configure a Multi Region Cluster by setting the broker.rack property on each host"

Sorry I don't mean to hijack this. I think the feature is called "multi region cluster"

Reply:
Thinking about it more dont we also need this one set for MRC?
replica.selector.class=org.apache.kafka.common.replica.RackAwareReplicaSelector

Might as well complete the loop and get that in somehow

Reply:
I'm good with updating the `replica.selector.class` as well.
I will also update the language a bit to be more generic as you stated.

I will then test it out on my AWS cluster to ensure they are set accordingly.

Reply:
NOTE: https://github.com/confluentinc/examples/blob/master/multiregion/README.md